request/dynamic_request.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+
+class Request:
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def dynamic_request(**kwargs: dict) -> str | dict:
+        method = kwargs.get('method', 'get').lower()
+        endpoint = kwargs.get('endpoint')
+        structure = kwargs.get('structure', 'json')
+
+        if not endpoint:
+            raise ValueError("Endpoint must be provided")
+        if not method:
+            raise ValueError(f"Invalid method: {method}")
+
+        kwargs.pop('method')
+        kwargs.pop('endpoint')
+        kwargs.pop('structure')
+
+        method = getattr(requests, method)
```

Comment:
A suggestion: instead of checking above for `if not method:` which protects against None value should we protect it against strings that don't correspond to request methods.  For example if `method="random"` this will raise an AttributeError. We could try and catch here and raise the Error nessage saying it is an invalid method

Reply:
```suggestion
        try:
                method = getattr(requests, method)
        except AttributeError:
                raise ValueError(f"Invalid method: {method}")
```